fn-fdk.js

```diff
@@ -463,6 +463,24 @@ class HTTPGatewayContext {
   }
 }
 
+/**
+ * TracingContext defines an OCI APM tracing context for the current invocation.
+ * Traces are currently defined by the Zipkin standard.
+ * See: https://zipkin.io/pages/instrumenting
+ */
+class TracingContext {
+  constructor (ctx) {
+    this.isEnabled = parseInt(ctx._config.OCI_TRACING_ENABLED) === 1
+    this.traceCollectorUrl = ctx._config.OCI_TRACE_COLLECTOR_URL
+    this.traceId = ctx.getHeader('X-B3-TraceId')
+    this.spanId = ctx.getHeader('X-B3-SpanId')
+    this.parentSpanId = ctx.getHeader('X-B3-ParentSpanId')
+    this.sampled = parseInt(ctx.getHeader('X-B3-Sampled')) === 1
+    this.flags = parseInt(ctx.getHeader('X-B3-Flags')) === 1
```

Comment:
The content of the flags header is currently only used for the debug flag but in theory could be used in the future for other things - we should make the content available verbatim rather than interpreting it as a boolean.

Reply:
Yeagh the only stuff I could really find was this: https://github.com/openzipkin/b3-propagation

We could leave it as a string and allow the user to interpret as they want. They are also free to use the main context and just fetch it off the header.

Reply:
The user can always grab it as a string from the context headers, but, yeagh if the other FDKs string it then happy to tweak it consistency is best.